src/components/Share/Share.scss

```diff
@@ -13,13 +13,21 @@
 
   .ta-mail-icon {
     a {
+      svg {
+        font-size: 23px;
+        margin-bottom: 2px;
+      }
+    }
+  }
+
+  .ta-draw-icon {
```

Comment:
The barriere free focus style is not perfect, yet. The box is a little bit to high.

Reply:
Fix see https://github.com/geops/trafimage-maps/pull/412